libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3358,6 +3358,7 @@ MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration_baro()
 
 MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration(const mavlink_command_long_t &packet)
 {
+    EXPECT_DELAY_MS(30000);
```

Comment:
Wow! Why does this only occur on some boards?

Reply:
Why put this here? Shouldn't the actual calibration methods be annotated appropriately?

Reply:
@WickedShell this covers all of them with one call, and the macro allows for both nesting and automatically unsets the timeout when the context is left. This simple approach ensures we don't miss one